src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -49,25 +49,31 @@
  *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
+ * @param {boolean} es5Out language out is ES5
  */
-$jscomp.inherits = function(childCtor, parentCtor) {
-  /** @constructor */
-  function tempCtor() {}
-  tempCtor.prototype = parentCtor.prototype;
-  childCtor.superClass_ = parentCtor.prototype;
-  childCtor.prototype = new tempCtor();
-  /** @override */
-    childCtor.prototype.constructor = childCtor;
+$jscomp.inherits = function(childCtor, parentCtor, es5Out) {
+  if (es5Out === true && Object.setPrototypeOf) {
```

Comment:
`setPrototypeOf` and `__proto__` weren't defined until ES2015. Ideally we could target ES5+ output for this, but I know of no way to do this.

IE 9 and 10 are considered ES5 compliant browsers. This is the crux of the difficulty.

Here's the state of things:

 * ES3 out (IE 8 and lower) -> `setPrototypeOf` can't be counted on. No ES3 browser supports this.
 * ES5 out (IE 9-11) -> `setPrototypeOf` works in IE11, but not in IE 9 or 10. `__proto__` only adds in an old set of Opera browsers.
 * ES6 out -> If only this worked ...

I recognize the desire to protect users, but in this case it's just not possible. Static inheritance doesn't properly work in IE 9 and 10.